.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,7 +11,7 @@
   - [ ] The new extension is used in hundreds of repositories on GitHub.com
     - Search results for each extension:
       <!-- Replace FOOBAR with the new extension, and KEYWORDS with keywords unique to the language. Repeat for each extension added. -->
-      - https://github.com/search?utf8=%E2%9C%93&type=Code&ref=searchresults&q=extension%3AFOOBAR+KEYWORDS+NOT+nothack
+      - https://github.com/search?q=path%3A*.FOOBAR+KEYWORDS+NOT+is%3Afork&type=code
```

Comment:
1.	I'd move `type=code` to the start of the query string, so that `q=path%3A*.FOOBAR+KEYWORDS` is less fiddly to edit (with `NOT is:fork` receiving similar treatment). So:
	```suggestion
	      - https://github.com/search?type=code&q=NOT+is%3Afork+path%3A*.FOOBAR+KEYWORDS
	```
2.	To match file extensions proper (as opposed to filenames that merely begin with a full-stop), the `path:` qualifier should be `**/?*.FOOBAR` instead. I'm not sure if this is even worth clarifying for users, so I'm including this as more of an FYI.

Reply:
Good idea about switching the order.

Re:

> the `path:` qualifier should be `**/?*.FOOBAR` instead.

I don't think this makes any difference when it comes to the new search. GitHub's own suggestion if you use the old `extension:foobar` is `path:*.foobar` and the [docs](https://docs.github.com/en/search-github/github-code-search/understanding-github-code-search-syntax) indicate this is sufficient too.

I've tested this with a few extensions and don't get different results. Do you have an example where you do?